UOP1_Project/ProjectSettings/ProjectSettings.asset

```diff
@@ -584,7 +585,7 @@ PlayerSettings:
   scriptingDefineSymbols: {}
   platformArchitecture: {}
   scriptingBackend:
-    Standalone: 1
+    Standalone: 0
```

Comment:
I draw **your** attention to the fact that I had to disable IL2CPP for the assembly. Otherwise, there will be an error that IL2CPP is not installed.